src/gluonts/mx/block/quantile_output.py

```diff
@@ -20,13 +20,46 @@
 from gluonts.mx import Tensor
 
 
+def uniform_weights(x: list) -> List[float]:
+    return [1.0 / len(x) for _ in x]
+
+
+def crps_weights_pwl(quantile_levels: List[float]) -> List[float]:
+    """
+    Computes the quantile loss weights corresponding to CRPS under linear
+    interpolation assumption.
+
+    Quantile levels are assumed to be sorted in increasing order.
+
+    CRPS = sum_{i=0}^{n-1} 0.5 * (q_{i+1} - q_{i}) * (z_{i+1} + z_{i})
```

Comment:
Can we use backticks (\`) so that the comment reads more nicely in the generated docs?

Reply:
I added directives for inline and displayed math throughout the docstring